hw/ip/otbn/dv/smoke/smoke_test.S

```diff
@@ -202,6 +202,25 @@ bn.addi w22, w21, 0x1bd
 # w23 = w21 - 0x207 = 0x78fccc06_2228e9d6_89c9b54f_887cf1df_df9bf9bd_f9bfd9ff_99ffbbbb_dbff99d4
 bn.subi w23, w21, 0x207
 
+# *x26(w24) = dmem[x25(0)] = 0xcccccccc_bbbbbbbb_aaaaaaaa_facefeed_deadbeef_cafed00d_d0beb533_1234abcd
```

Comment:
I did initially go with that style but decided that was confusing because it overloads the meaning of '=' namely between using it for both assignment (make LHS take value of RHS) and equality (LHS is RHS). So in `*x26 = w24 = dmem...` we're saying w24 is the contents of x26 and we're putting the value of `dmem...` in w24.

So I went with this to avoid that confusion. I could instead use `:=` and `=` for instance to avoid overloading the meaning of '=' or '=' and '=='.

Reply:
Fine with me either way, it's the =/== distinction is used already further down (in a later commit, IIRC).

Reply:
Could you provide an example of what you want for documenting intermediate values? I can't quite see what you mean there.

Reply:
I was thinking about things like the value of x26 before the instruction is executed, but looking at it again, I don't find the specific case any more that prompted me to write this comment, and the information "x26 has the value 24" is present in the "x26 =(=) w24" nomenclature. It could be more obvious by having an additional line "x26 == 24", but that's not strictly necessary.